src/Tgstation.Server.Host/Core/ServerAddressProvider.cs

```diff
@@ -41,23 +45,14 @@ sealed class ServerPortProivder : IServerPortProvider
 				.GetSection("Http")
 				.GetSection("Url")
 				.Value;
-
-			if (generalConfiguration.ApiPort == default && httpEndpoint == null)
-				throw new InvalidOperationException("Missing required configuration option General:ApiPort!");
-
-			if (generalConfiguration.ApiPort != default)
-				return;
-
-			logger.LogWarning("The \"Kestrel\" configuration section is deprecated! Please set your API port using the \"General:ApiPort\" configuration option!");
-
-			var splits = httpEndpoint.Split(":", StringSplitOptions.RemoveEmptyEntries);
-			var portString = splits.Last();
-			portString = portString.TrimEnd('/');
-
-			if (!UInt16.TryParse(portString, out var result))
-				throw new InvalidOperationException($"Failed to parse HTTP EndPoint port: {httpEndpoint}");
-
-			generalConfiguration.ApiPort = result;
+			if (httpEndpoint != null)
+				throw new InvalidOperationException("The old Kestrel configuration section is no longer supported, you must remove it. Please set your API port and bind address using the \"General:ApiPort\" \"General:BindAddress\" configuration options!");
```

Comment:
If we're finally fully dropping support for it, we'll need to do a major version bump for the config version in `build/Version.props`

Reply:
Also, since we're doing a major bump, feel free to change the default bind address to not be on 0.0.0.0 if you think it's worth it.

Reply:
ok I will do that